code/stable/ssp/Website/SSP_SRS.html

```diff
@@ -1055,7 +1055,7 @@ <h3>Goal Statements</h3>
                   </p>
                   <p>
                     <div id="determineShearF">
-                      Determine-Shear-Forces: Determine the interslice shear forces between each pair of vertical slices of the slope.
+                      Determine-Shear-Forces: Determine the interslice normal and shear force proportionality between each pair of vertical slices of the slope.
```

Comment:
This change is not correct.  The goals for determining the shear force and the normal force are separate goals.